uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -279,6 +276,14 @@ private static function validateMustTakePublicHolidayAsLeave($params) {
    * @throws \CRM_HRLeaveAndAbsences_Exception_InvalidAbsenceTypeException
    */
   private static function validateAbsenceTypeTitle($params) {
+    $isEditing = array_key_exists('id', $params);
+    $titleIsPassed = array_key_exists('title', $params);
+    $skipTitleValidation = $isEditing && !$titleIsPassed;
+
+    if ($skipTitleValidation) {
+      return;
+    }
```

Comment:
What about reducing this to a few lines ?
```php
if (!empty($params['id'] && !isset($params['title']) {
  return;
}

Reply:
I would prefer understanding the code better rather having 2 lines saved, as a developer, especially this `!empty($params['id'])` - developers may forget why we are checking the ID.

Also I would prefer `array_key_exists` because this function represents exactly what I want to introduce in this logic (see https://stackoverflow.com/questions/3210935/difference-between-isset-and-array-key-exists)

Reply:
Well you have a point for making the code more readable.